docs/result.md

```diff
@@ -227,3 +227,156 @@ File: document.json
+### Docket:
+
+1- bad docketID:
+
+ docketID="EPA-HQ-OAR-2011-0028-0000"
+
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/
+      regulations/v3/docket.json?
+      api_key="VALID KEY"&docketID=
+      EPA-HQ-OAR-2011-0028-0000'
+    :received 404:Not Found'
+    }
+```
+
+2-wrong docketID pattern:
+
+ docketID=ASD-EPA-HQ-OAR-2011-0028-DDD
+
+
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/
+      regulations/v3/docket.json?
+      api_key="VALID KEY"&docketID=ASD-EPA-HQ-OAR-2011-0028-DDD'
+      :received 400:Bad request'
+    }
+```
+3- bad api kay :
+
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/
+      regulations/v3/docket.json?
+      api_key="INVALID"&docketID=ASD-EPA-HQ-OAR-2011-0028-DDD'
+      :received 403:Forbidden'
+    }
+```
+
+### Document:
+
+1- bad documentID:
+ documentID="EPA-HQ-OAR-2011-0028-0108-0000"
+
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/
+      regulations/v3/docket.json?
+      api_key="VALID KEY"&documentID=
+      EPA-HQ-OAR-2011-0028-0108-0000'
+    :received 404:Not Found'
+    }
+```
+2- Overused api key:
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/regulations/v3/document.json?api_key=VALID KEY"&po=1000&crd=11/06/13 - 03/06/14'
+    :received 429:Too Many Requests'
+    }
+```
+
+
+
+### Documents:
+1- Bad URL:
+
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/regulations/v3/document.json?api_key=VALID KEY"&po=1000&crd=11/06/13 - 03/06/14'
+    :received 404:Not Found'
+    }
+```
+2- No api key:
+
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/regulations/v3/documents.json?api_key="&po=1000&crd=11/06/13 - 03/06/14'
+    :received 403:Forbidden'
+    }
+```
+
+3-Overused api key:
+
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/regulations/v3/documents.json?api_key=VALID KEY"&po=1000&crd=11/06/13 - 03/06/14'
+    :received 429:Too Many Requests'
+    }
+```
+
+### Errors connections:
+
+######1-  500 Internal Server Error: 
+it happend when the client try conecte to the server but the srever cannot process the request for an unknown reason.
+
+
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Message':''URL:https://api.data.gov:443/regulations/v3/documents.json?api_key=VALID KEY"&po=1000&crd=11/06/13 - 03/06/14'
+    :received 500:Internal Server Error'
+    }
+```
+
+######2-  503 Service Unavailable:
+it happend when the client try to conecte to the server but the server is overloaded or under maintenance.
```

Comment:
In a few spots there is the spelling error for 'happened' and 'connect'.
Maybe also reword it so instead of 'It happened' we state 'Occurs when...'